src/sentry/integrations/request_buffer.py

```diff
@@ -51,10 +87,15 @@ def is_integration_broken(self):
                 and int(day_count.get("success_count", 0)) == 0
             ):
                 days_error.append(day_count)
+            elif int(day_count.get("timeout_count", 0)) >= settings.BROKEN_TIMEOUT_THRESHOLD:
+                days_timeout.append(day_count)
 
         if len(days_fatal) > 0:
             return True
 
+        if len(days_timeout) > 0:
+            return True
```

Comment:
Not sure if this is a big deal, but I triggered disabling it locally, re-subscribed to the webhook and triggered _one_ timeout error, and it disabled it again. We should probably clear the buffer when we disable in case someone like me tries to fix it on the same day.